### PR TITLE
HOCS-1838: Apply responsive css to specific components.

### PR DESCRIPTION
**Pull request by parth-ho**
Applied responsive css to three entity list components: 1.Correspondents, 2.DraftDocuments, 3.Topics.

### CODE REVIEW

File: src/styles/app.scss [hunk 1]
```diff
@@ -761,19 +761,22 @@ $border-colour: govuk-colour("grey-2");
...
-      table-layout: fixed;
-    }
-  }
-}
+#Correspondents, #DraftDocuments, #Topics {
+  table {
+    table-layout: fixed;
+    overflow-wrap: break-word;
+
+    > tbody > tr > td {
```
Comment (zuzak) [inline src/styles/app.scss:769]: Would just `td` be sufficient? (And more efficient)

File: src/styles/app.scss [hunk 2]
```diff
@@ -381,7 +381,7 @@ a.card__body {
             padding: 0.25em 0.5em 0.25em 0.5em;
           }
       }
-      &.indicator {          
+      &.indicator {
```
Comment (zuzak) [inline src/styles/app.scss:384]: Ideally these whitespace fixes would be in a separate commit

File: src/styles/app.scss (hunk 1 above)
Comment (Paulcr-ho) [reply src/styles/app.scss:769]: Verified #Correspondents, #DraftDocuments, #Topics against the form data (loaded in hocs-data project).
It all looks good to me. The govuk-form-group and govuk-table__cell are placed here by the new formatting. 
There is no danger of affecting other components or formatting as they were both inserted in the css for this ticket.
